Libraries/Utilities/BackAndroid.android.js

```diff
@@ -42,6 +42,10 @@ RCTDeviceEventEmitter.addListener(DEVICE_BACK_EVENT, function() {
  * Example:
  *
  * ```js
+ * const ReactNative = require('react-native');
+ * const {
+ *   BackAndroid
+ * } = ReactNative;
```

Comment:
I think it's better use ES6 syntax, like this?

`import {BackAndroid} from 'react-native';`